source/faucet/main.d

```diff
@@ -282,6 +283,71 @@ public class Faucet : IFaucet
                    .map!(kv => tuple(kv.key, kv.value))
                    .assocArray();
     }
+
+    /// POST: /send_transaction
+    public override void sendTransaction (string recv, ulong amount)
+    {
+        PublicKey pubkey = PublicKey.fromString(recv);
+        Amount leftover = Amount(amount);
+        auto first_utxo = this.getOwnedUTXOs().byKeyValue().take(1);
```

Comment:
If you are only interested in one utxo, why use `take(1)`? You could just use `.byKeyValue().front` and then `first_utxo` will become a tuple instead of a range with one element of a tuple.

Then you wouldn't need to use `map` in any of the calls below.